trillian/ctfe/configpb/config.proto

```diff
@@ -47,4 +63,7 @@ message LogConfig {
   // accept_only_ca controls whether or not *only* certificates with the CA bit
   // set will be accepted.
   bool accept_only_ca = 10;
+  // backend_name if set indicates which backend serves this log. The name must be
```

Comment:
"if set": are you planning on keeping support for both this and `rpcBackendFlag`?

Reply:
I talked to Pierre and we decided that that flag would remain for the simple case so existing configs don't break. If you don't set it then the config in this file must be present.